styles.css

```diff
@@ -0,0 +1,11 @@
+@media only screen and (max-width: 800px) {
+    body {
+      background-color: lightpink;
+    }
+  }
+  
+  #main-doc{
+    position:reltive;
+    margin-left:0px;
+  margin-top:300px;
+  }
```

Comment:
Once your `<head>` section is changed, you will need to link your style.css sheet and index.html with a **rel attribute** inside of it (the `<head>` tag) so the changes will show.